tox.ini

```diff
@@ -0,0 +1,66 @@
+[tox]
+downloadcache = {toxworkdir}/_download/
+envlist = py27-dj{14,18},coverage,pep8,pylint,pyflakes
+
+[testenv]
+commands = {envpython} manage.py test
+
+[testenv:py27-dj14]
+deps =
+    django == 1.4.22
+    edx-opaque-keys
+    mock
+    django_nose>=1.4
+
+[testenv:py27-dj18]
+deps =
+    django >= 1.8, < 1.9
+    edx-opaque-keys
+    mock
+    django_nose>=1.4
+
+[testenv:pep8]
+deps = pep8
+commands = {envbindir}/pep8 submit_and_compare/
+
+[testenv:pylint]
+deps = pylint
+commands = {envbindir}/pylint submit_and_compare/
+
+[testenv:pyflakes]
+deps = pyflakes
+commands = {envbindir}/pyflakes submit_and_compare/
+
+[testenv:coverage]
+deps =
+    coverage
+    django == 1.4.22
+    edx-opaque-keys
```

Comment:
We should be able to move most of these requirements to `setup.py` (`install_requires`)
and then not redefine them in `tox.ini`:
- django
- edx-opaque-keys
- mock
- django_nose
- coverage

`coveralls` can be explicitly added to the `coveralls` testenv,
since that's the only time we'll ever need it.

`django >= 18, < 1.9` will also need to be redefined in-line.